Mention manual triggering as an option (#132)
ca-johnson/perforce-buildkite-plugin: README.md

## Triggering Builds

There are a few options for triggering builds that use this plugin, in this order from least valuable but most convenient to most valuable but least convenient.

### Manual

Relies on people within your team manually clicking `New Build` within the BuildKite UI.

> * To build current head of the stream - accept the defaults.

Reviewer: ```suggestion
* To build current head revision on the server - accept the defaults.
```

> * To build a specific revision - paste the revision number into the `Revision` textbox.

Reviewer: ```suggestion
* To build a specific revision - paste the revision number into the `Commit` textbox.
```

> * To build a Shelf Set (pending Change List) - paste your revision to the `Branch` textbox.

Reviewer: ```suggestion
* To build a shelved changelist - paste your changelist number into the `Branch` textbox.
```

### Schedule

> Schedule builds with a cron in buildkite - this requires no additional setup, but provides the worst reponse time as changes are made

Reviewer: ```suggestion
Schedule builds with a cron in buildkite - this requires no additional setup, but provides the worst response time as changes are made
```

Oops, old typo
